tests/playbooks/centos7_deps.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install CentOS deps for network system role tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+        - git
+        - python3-pip
+        - python2-pip
```

Comment:
Why python 2 and python 3?

Reply:
I was thinking of using this playbook to load the CentOS7 python 2 dependencies too. It might be too much of a mixture though.

Reply:
Please remove python2-pip for now.